hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMaster.java

```diff
@@ -2699,11 +2701,11 @@ public void checkTableModifiable(final TableName tableName)
     }
   }
 
-  public ClusterMetrics getClusterMetricsWithoutCoprocessor() throws InterruptedIOException {
-    return getClusterMetricsWithoutCoprocessor(EnumSet.allOf(Option.class));
+  public ClusterMetrics getClusterMetricsInternal() throws InterruptedIOException {
+    return getClusterMetricsInternal(EnumSet.allOf(Option.class));
   }
 
-  public ClusterMetrics getClusterMetricsWithoutCoprocessor(EnumSet<Option> options)
+  public ClusterMetrics getClusterMetricsInternal(EnumSet<Option> options)
```

Comment:
This changes the public method name on an LP(TOOLS) class in a minor release. Are there any tools not in the immediate project that depend on this?

Reply:
This isn't a required change, I just did it because everywhere else in the code the convention is fooInternal, not fooWithoutCoprocessor, and I find it weird and worth changing. I doubt there are any TOOLS that require this method but this change isn't necessary either.